extensions/wikia/Email/Controller/FounderDigestController.class.php

```diff
@@ -131,9 +134,9 @@ protected function assertSubscribedToCompleteDigest() {
 			$this->wikiId
 		);
 
-		if ( !$wantsCompleteDigest ) {
-			throw new Check( 'Founder is not subscribed to complete digest.' );
-		}
+//		if ( !$wantsCompleteDigest ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)